pinot-spi/src/test/java/org/apache/pinot/spi/utils/JsonUtilsTest.java

```diff
@@ -641,4 +641,71 @@ public void testEmptyString()
     List<Map<String, String>> flattenedRecords = JsonUtils.flatten(jsonNode, jsonIndexConfig);
     assertTrue(flattenedRecords.isEmpty());
   }
+
+  @Test
+  public void testFlattenWithIndexPaths()
+      throws IOException {
+    {
+      JsonNode jsonNode = JsonUtils.stringToJsonNode(
+          "[{\"country\":\"us\",\"street\":\"main st\",\"number\":1},{\"country\":\"ca\",\"street\":\"second st\","
```

Comment:
Is there anyway we can make this json string easier to read? Ideally without escape so that it is easier to verify the depth?

Reply:
Added the better view as comments. The code is using the same style as the existing ones.

Reply:
Something to consider: maybe this could benefit from abstracting out the JSON strings from the java file into a separate test JSON or text resource file.